### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by mford008**
Centered hero content, filled content below hero, created new scss document for content below hero - styling WIP

### CODE REVIEW

**Comment by adborden** [comment]
@mford008 if this is WIP, mind if we add that in the PR title so it's clear?

**Comment by mford008** [comment]
Absolutely, WIP added. Thanks.

**Comment by adborden** [comment]
Fixes #116

**Comment by mford008** [comment]
Left to do: 1. pull in "funds raised" totals from back end data, 2. get "funds raised" numbers to render as gold, 3. final spacing alterations.

**Comment by adborden** [comment]
This looks great, but we can't ship it with the placeholder numbers. We should probably remove them for now, and then add them back in another PR. Sound good?

**Comment by mford008** [comment]
Sounds good to me, thanks!

File: _sass/module/_hero.scss [hunk 1]
```diff
@@ -44,7 +46,6 @@ $hero-breakpoint: $grid--medium;
     &::before {
       content: '';
       position: absolute;
-      background-color: $scrim-color;
```
Comment (adborden) [inline _sass/module/_hero.scss]: We probably just want to delete this whole pseudo element. It's only purpose was to provide the scrim.

File: _sass/module/_hero.scss [hunk 2]
```diff
@@ -62,10 +63,15 @@ $hero-breakpoint: $grid--medium;
 // Layout element
 .hero__main {
   @include grid-column(12);
+  font-weight: 350;
+  font-style: italic;
+  display: flex;
```
Comment (adborden) [inline _sass/module/_hero.scss]: I'm not really familiar with flexbox, but the grid is based on floats and margins, so I'm not sure we want to mix them. You probably want to remove the `@include grid-column`, but I'm not sure.

File: _sass/module/_landing-description.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.landing__description__header {
+  @include heading('h3');
+  color: $color-grey-2;
```
Comment (adborden) [inline _sass/module/_landing-description.scss]: I moved this to the `heading()` mixin, I think all `h3`'s should be this color, right?

File: _sass/module/_landing-description.scss [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  font-style: italic;
+  font-weight: 400;
+}
+
+.landing__description {
+  padding: 50px;
```
Comment (adborden) [inline _sass/module/_landing-description.scss]: This is proabably too much padding on mobile.

File: _sass/module/_hero.scss [hunk 5]
```diff
@@ -24,7 +24,7 @@ $hero-breakpoint: $grid--medium;
 
 .hero--photo {
   @include media($hero-breakpoint) {
-    background-image: url('./images/sacramento-capitol-splash.jpg');
```
Comment (adborden) [inline _sass/module/_hero.scss]: We can delete this file too.

**Comment by adborden** [comment]
Besides removing the placeholder content, I tweaked the color variables a bit since they seemed to be affecting more than just the landing page and hero. I also looked at the changes on mobile and to fix a few things for it to look okay. We should probably go over some of my comments together :)

**Pull request merged**